### PR TITLE
Fixed so that you can upload pictures.

### LINKED ISSUE #1: Enable image uploads

### PR DESCRIPTION
**Pull request by Adasjo**
I think this will enable upload with pictures.

### CODE REVIEW

**Review by Abendrill — CHANGES_REQUESTED** [review]
I see the appeal of enabling uploads. The reality of the situation is however more complex than flipping boolean flag.

Our systems are deployed in ephermal docker containers. Any modifications of the filesystems in the container, e.g. the PHP app saving the uploaded file, will be lost upon restart.

A solution for maintanable persistent storage is therefore unfortunately needed before we can enable image uploads.

**Comment by DokID** [comment]
Put in layman's terms, you need to somehow map the folder that the images are saved in in the container to an actual, physical location on the host machine. In Docker, this entails creating a so-called volume, however, as far as I know, none of our projects utilize volumes this way, since most of our systems save their data to a database of some kind, which in turn is persistent.

Since we currently rely completely on Dokku (our hosting platform) auto-importing the Dockerfiles used for building the containers our apps are run in, adding a volume can be tricky, since adding our own Dockerfile will make Dokku use that for building, rather than fetching an appropriate one from its own repositories. 

I guess maybe you could either fork a buildpack Dockerfile and add it to the repo with an additional volume definition, or perhaps create a local Dockerfile, import the desired buildpacks dockerfile in that, and then add a volume definition. This, however, is purely speculative... I am not sure how Dokku would want us to deal with this.

**Comment by DokID** [comment]
Ok, so after some quick googling I found a couple of ways this could be done, but I don't really have time to take care of this right this second, so I am going to have to get back to this in a few days maybe. I also want to take a look at how much space our servers even have, and if we could maybe mount some S3 storage volume in AWS to it so that we don't have a pile of images stacking up on our poor, fragile and rather crucial servers.

Could someone figure out where the images are saved in the container?

**Comment by addreas** [comment]
It's been long since I touched dokku, but [this](https://github.com/dokku/dokku/blob/master/docs/advanced-usage/persistent-storage.md) seems to be perfectly reasonable as far as I can tell. Shouldn't require any Dockerfile trickery at all, since volumes can be specified at runtime. 

But perhaps it is easier to use a [mediawiki extension](https://www.mediawiki.org/wiki/Extension:AWS )?

**Comment by DokID** [comment]
Yeah I also saw that, and it's not that it's technically hard, I just would prefer finding a sustainable (ish) solution to begin with rather than doing what for example META-TV did (at least I think it was patched :thinking:) which was to endlessly pile up data (history was never cleared) on an actual production machine until the whole thing finally crashed.

**Comment by Abendrill** [comment]
Maybe there exists an S3 plugin for MediaWiki?

**Comment by Abendrill** [comment]
Like https://m.mediawiki.org/wiki/Extension:AWS

I've not wetted it at all, it was just the first thing I found when I searched for it
